docs/documentation/securing_apps/topics.adoc

```diff
@@ -8,19 +8,8 @@ include::topics/overview/terminology.adoc[]
 
 include::topics/oidc/oidc-overview.adoc[]
 
-include::topics/oidc/available-endpoints.adoc[]
-
-include::topics/oidc/supported-grant-types.adoc[]
-
 include::topics/oidc/oidc-errors.adoc[]
```

Comment:
This chapter `oid-errors` should also be included, right?

Reply:
done! should we do the same with saml-errors.adoc?

Reply:
Yep, we should move the saml error chapter to somewhere in the end. Probably to saml galleon chapter. But I think in this one it was clear that it should be maintained here.